## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 536067 --additional-package podman.tests`
Commit: `d7a368779abec248368e8c0129854b168ed1e68d`

---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>glances</li>
    <li>podman.tests.podman-tls-ghostunnel</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 15 packages built:</summary>
  <ul>
    <li>ansible-builder (python313Packages.ansible-builder)</li>
    <li>ansible-navigator</li>
    <li>apx</li>
    <li>apx-gui</li>
    <li>out-of-tree</li>
    <li>podman</li>
    <li>podman-desktop</li>
    <li>podman.tests.oci-containers-podman</li>
    <li>podman.tests.oci-containers-podman-rootless-conmon</li>
    <li>podman.tests.oci-containers-podman-rootless-healthy</li>
    <li>podman.tests.podman</li>
    <li>python314Packages.ansible-builder</li>
    <li>ramalama</li>
    <li>sh4d0wup</li>
    <li>snouty</li>
  </ul>
</details>


---
<details>
<summary>glances</summary>
<pre>
/nix/store/lkiz1ls5ivhi5dzw0nx0k0xsjcaklm4j-python3.13-requests-2.33.1/lib/python3.13/site-packages/requests/adapters.py:678: ConnectionError
----------------------------- Captured stdout call -----------------------------

==============================================================================
INFO: [TEST_001] Get all stats
HTTP RESTful request: http://localhost:61234/api/4/all
--------------------------- Captured stderr teardown ---------------------------
INFO:     Started server process [3509]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:61234 (Press CTRL+C to quit)
=========================== short test summary info ============================
FAILED tests/test_browser_restful.py::TestServersListEndpoint::test_serverslist_returns_200 - requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, p...
FAILED tests/test_browser_restful.py::TestServersListEndpoint::test_serverslist_returns_list - requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, p...
FAILED tests/test_browser_restful.py::TestServersListEndpoint::test_serverslist_has_servers - requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, p...
FAILED tests/test_browser_restful.py::TestServersListEndpoint::test_serverslist_server_has_required_fields - requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, p...
FAILED tests/test_browser_restful.py::TestServersListEndpoint::test_serverslist_server_types - requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, p...
FAILED tests/test_restful.py::TestGlances::test_001_all - requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, p...
===== 6 failed, 611 passed, 14 skipped, 16 deselected in 102.94s (0:01:42) =====</pre>
</details>

--- a/nixos/tests/podman/tls-ghostunnel.nix
+++ b/nixos/tests/podman/tls-ghostunnel.nix
@@ -83,7 +83,7 @@ import ../make-test-python.nix (
     };
 
     testScript = ''
-      import os
+      import subprocess
       import shlex
 
 
@@ -93,7 +93,7 @@ import ../make-test-python.nix (
 
       def cmd(command):
         print(f"+{command}")
-        r = os.system(command)
+        r = subprocess.run(command, shell=True).returncode
         if r != 0:
           raise Exception(f"Command {command} failed with exit code {r}")
 

--- a/pkgs/by-name/po/podman/package.nix
+++ b/pkgs/by-name/po/podman/package.nix
@@ -204,5 +204,6 @@ buildGoModule (finalAttrs: {
     license = lib.licenses.asl20;
     teams = [ lib.teams.podman ];
     mainProgram = "podman";
+    platforms = lib.platforms.linux;
   };
 })

